website/src/pages/api/admin/users.ts

```diff
@@ -1,11 +1,12 @@
 import { withRole } from "src/lib/auth";
 import { oasstApiClient } from "src/lib/oasst_api_client";
 import prisma from "src/lib/prismadb";
+import { BackendUser } from "src/types/Users";
 
 /**
  * The number of users to fetch in a single request.  Could later be a query parameter.
  */
-const PAGE_SIZE = 20;
+const PAGE_SIZE = 1;
```

Comment:
Is this part intentional? Or should we switch it back to 20?

Reply:
Sorry, I was debugging the pagination. Fixed with latest commit